modules/tensor_mechanics/test/tests/dynamics/wave_1D/wave_hht.i

```diff
@@ -24,229 +24,125 @@
   ymax = 4.0
   zmin = 0.0
   zmax = 0.1
+  use_displaced_mesh = false
 []
 
-
-[Variables]
-  [./disp_x]
-  [../]
-  [./disp_y]
-  [../]
-  [./disp_z]
-  [../]
-[]
-
-[AuxVariables]
-  [./vel_x]
-  [../]
-  [./accel_x]
-  [../]
-  [./vel_y]
-  [../]
-  [./accel_y]
-  [../]
-  [./vel_z]
-  [../]
-  [./accel_z]
-  [../]
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
 []
 
-[Kernels]
-  [./DynamicTensorMechanics]
-    displacements = 'disp_x disp_y disp_z'
-    alpha = -0.3
-  [../]
-  [./inertia_x]
-    type = InertialForce
-    variable = disp_x
-    velocity = vel_x
-    acceleration = accel_x
-    beta = 0.3025
-    gamma = 0.6
+[Modules/TensorMechanics/DynamicMaster]
+  [all]
+    add_variables = true
```

Comment:
Can you add stiffness and mass damping to the dynamic master action and a test for that (preferably large deformation)?

Reply:
The `eta` parameter? Yes, can do.

Reply:
Yeah, `eta` could be something like `mass_damping_coefficient` and `zeta` something like `stiffness_damping_coefficient` for the action user.